living_documentation_regime/action_inputs.py

```diff
@@ -172,4 +179,16 @@ def validate_user_configuration(self) -> None:
                 )
                 sys.exit(1)
 
+        # log user configuration
         logger.debug("User configuration validation successfully completed.")
+        logger.debug("User required input `liv-doc-regime`: %s.", ActionInputs.get_liv_doc_regime())
+        logger.debug("User required input `output-formats`: %s.", ActionInputs.get_output_formats())
+        logger.debug("User input `report-page`: %s.", ActionInputs.get_is_report_page_generation_enabled())
+        logger.debug("User regime input `liv-doc-repositories`: %s.", ActionInputs.get_repositories())
+        logger.debug(
+            "User input `liv-doc-project-state-mining`: %s.", ActionInputs.get_is_project_state_mining_enabled()
```

Comment:
This should be conditional right?

Reply:
Sorry. Don't understand your question.

Reply:
There is another idea, that `action_inputs.py` should not be in a `living_documentation_regime` folder, since they are going to store all the inputs from all regimes. Shouldnt it?

Reply:
Correct. The file has been moved in commit a4b7fd0370176540ded3f10755d26a0329cdd432.